### DISCUSSION TITLE
Switch from different checkboxes

### DISCUSSION THREAD

**Discussion by smyja** [mantinedev/mantine#2263] [Q&A] [answered]
For the checkbox group, I'd like to create one check at a time like in quizzes. Someone suggested I create a state for each box, Is there another way to do it?

**Comment by rtivital** [comment, marked as answer]
Checkbox are for multiple selections, if you want only one selection to be active at a time, use Radio component. If you still want to use Checkbox for that, then you should be able to implement it with controlled Checkbox.Group.
